Sources/TripKitUI/views/cell accessory/QuantityStepperAccessoryView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  QuantityStepperAccessoryView.swift
+//  TripKitUI-iOS
+//
+//  Created by Jules Gilos on 7/27/22.
+//  Copyright © 2022 SkedGo Pty Ltd. All rights reserved.
+//
+
+import UIKit
+
+import RxSwift
+
+class QuantityStepperAccessoryView: UIView {
```

Comment:
As discussed separately, this isn't needed in this repo, and can be just in the TripGo repo.